x/msgfees/keeper/keeper.go

```diff
@@ -71,6 +73,15 @@ func (k Keeper) GetFloorGasPrice(ctx sdk.Context) sdk.Coin {
 	return min
 }
 
+// GetUsdConversionRate returns the current usd to hash (1000000000nhash) conversion rate in mils
```

Comment:
A rate is a fraction. So that needs to be part of the units listed.
```suggestion
// GetUsdConversionRate returns the current usd to hash (1000000000nhash) conversion rate in mils per hash.
```